services/tenant-ui/frontend/src/store/governanceStore.ts

```diff
@@ -8,20 +8,91 @@ export const useGovernanceStore = defineStore('governance', () => {
   const schemaTemplates: any = ref(null);
   const selectedSchemaTemplate: any = ref(null);
   const schemaTemplateFilters: any = ref(null);
+  
+  const credentialTemplates: any = ref(null);
+  const selectedCredentialTemplate: any = ref(null);
+  const credentialTemplateFilters: any = ref(null);
 
   const loading: any = ref(false);
   const error: any = ref(null);
 
   // getters
+  const schemaLabelValue = (item: any) => {
+    let result = null;
+    if (item != null) {
+      result = { 
+        'label': `${item.name} (${item.version})`, 
+        'value': item.schema_template_id,
+        'status': item.status,
+        'can_issue': item.credential_templates.length > 0
+      }
+    }
+    return result;
+  };
+
+  const schemaTemplateDropdown = computed(() => {
+    //transform list of schemas into label / value pair
+    let result: any[] = [];
+    if (schemaTemplates.value != null) {
+      result = schemaTemplates.value
+        .map((item: any) => schemaLabelValue(item));
+    }
+    return result;
+  });
```

Comment:
Awesome, almost the exact same code I have locally for the contact dropdown getter (might be able to have some more general utility that maps the dropdown but that could be more work than it's worth since the field names will be specific)